examples/SimpleTempSensor/SimpleTempSensor.ino

```diff
@@ -0,0 +1,45 @@
+/*
+  Arduino LSM6DS3 - Simple Gyroscope
+
+  This example reads the temperature values from the LSM6DS3
+  sensor and continuously prints them to the Serial Monitor
+  or Serial Plotter.
+
+  The circuit:
+  - Arduino Uno WiFi Rev 2 or Arduino Nano 33 IoT
+
+  created 15 Mar 2020
+  by Alex Hoppe
+
+  This example code is in the public domain.
+*/
+
+#include <Arduino_LSM6DS3.h>
+
+void setup() {
+  Serial.begin(9600);
+  while (!Serial);
+
+  if (!IMU.begin()) {
+    Serial.println("Failed to initialize IMU!");
+
+    while (1);
+  }
+
+  Serial.print("Temperature sensor sample rate = ");
+  Serial.print(IMU.tempSampleRate());
+  Serial.println(" Hz");
+  Serial.println();
+  Serial.println("Temperature sensor in degrees C");
+  Serial.println("T");
+}
+
+void loop() {
+  float t;
```

Comment:
I think it would be nice to have a comment explaining the purpose of this variable. Beginners might expect that `IMU.readTemp()` will return the temperature reading, and not be familiar with the concept of passing arguments by reference.

Reply:
Good idea, I'm just not sure how I would word it. If you have a suggestion I could implement it in the other two examples too.

Reply:
I decided the better place for the explanatory comment would be on the `IMU.readTemperature()` call. I submitted a suggestion for this on that line.

>I could implement it in the other two examples too

I think that would be great, but please do it in a separate pull request to keep this pull request atomic.